sandbox/pages/HomePage.vue

```diff
@@ -81,23 +83,45 @@ const filteredComponents = computed(() => {
       type: 'multi-color',
       name: key,
       component: val,
+      keyword: '',
```

Comment:
I suggest we name it plural (`keywords`) and make it an array. Some other icons could benefit from having multiple search terms defined as keywoards (e.g. CogIcon -> `keywoards: ['gear', 'settings']`). I don't ask you to add keywords to that icon - I just use it as an example as to why array would be more beneficial